packages/components/atoms/f-filter-pill/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@justeat/f-filter-pill",
+  "description": "Fozzie Filter Pill - An interactive component for filter toggles.",
+  "version": "0.1.0",
+  "main": "dist/f-filter-pill.umd.min.js",
+  "files": [
+    "dist",
+    "test-utils"
+  ],
+  "homepage": "https://github.com/justeat/fozzie-components/tree/master/packages/components/atoms/f-filter-pill",
+  "contributors": [
+    "Github contributors <https://github.com/justeat/fozzie-components/graphs/contributors>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:justeat/fozzie-components.git"
+  },
+  "bugs": {
+    "url": "https://github.com/justeat/fozzie-components/issues"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "keywords": [
+    "fozzie"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "build": "vue-cli-service build --target lib --name f-filter-pill ./src/index.js",
+    "lint": "vue-cli-service lint",
+    "lint:fix": "yarn lint --fix",
+    "lint:style": "vue-cli-service lint:style",
+    "test": "vue-cli-service test:unit",
+    "test-component:chrome": "cross-env-shell TEST_TYPE=component wdio ../../../../wdio-chrome.conf.js",
+    "test-a11y:chrome": "cross-env-shell TEST_TYPE=a11y wdio ../../../../wdio-chrome.conf.js",
+    "test:visual": "cross-env-shell PERCY_TOKEN=${PERCY_TOKEN_F_FILTER_PILL} TEST_TYPE=visual percy exec -- wdio ../../../../wdio-chrome.conf.js"
+  },
+  "browserslist": [
+    "extends @justeat/browserslist-config-fozzie"
+  ],
+  "dependencies": {
+    "@justeat/f-services": "1.7.0"
```

Comment:
Do you use this dep?

Reply:
not sure yet. I'll remove it for now, and add it later if we need it.